torch/_higher_order_ops/utils.py

```diff
@@ -525,6 +527,42 @@ def diff_tensor_meta(
     return pair_diffs
 
 
+# Create a new code object with a different filename to bypass skipfile check
+def _make_inlinable(f: Callable) -> Any:
```

Comment:
This is a hacky way of doing it. Maybe we should have a _mark_inlinable flag or something to bypass the skip file check.

Reply:
all of torch/_higher_order_ops should not be in SkipFiles

Reply:
yeah, control_flow ops are not in skipfiles. The issue here is that we create a local function in  torch/_dynamo/variables/higher_order_ops.py, which is in skip files. We could have a dedicated file under torch/_higher_order_ops to store all these temporary functions for subgraphs but this scatters the dynamo implementation into several places, which is not ideal. Be able to define local functions seems UX.

Reply:
We should have a dedicated file under torch/_dynamo/variables where we inline the entire file. torch/_dynamo/variables/inlineable_hop_utils.py

Reply:
My worry about make_inlineable is I'm not sure how predictable it is. If it calls out to another function, does that need to be marked as make_inlineable as well?